### ISSUE TITLE
[dev.icinga.com #6615] Perfdata values are normalized on base UOM

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#1747]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/6615

**Created by ccesario on 2014-07-01 01:32:21 +00:00**

Assignee: _(none)_
Status: _Rejected_  (closed on _2014-07-11 07:53:49 +00:00_)
Target Version: _(none)_
Last Update: _2014-09-27 00:00:26 +00:00 (in Redmine)_

    Icinga Version: v2.0.0-31-g99b590a

---

Is there any reason to perfdata values to be converted by icinga2? This value doesn't should remain untouched?

The check\_ping rta perfdata is converted from ms to sec.

    [2014-06-27 17:01:11 -0300] debug/IdoMysqlConnection: Query: UPDATE icinga_servicestatus SET acknowledgement_type = '0',  active_checks_enabled = '1',  check_command = 'ping4',  check_source = 'icinga-master',  check_type = '0',  current_check_attempt = '1',  current_notification_number = '0',  current_state = '0',  endpoint_object_id = 2,  event_handler = '',  event_handler_enabled = '0',  execution_time = '4.0375831127166748',  flap_detection_enabled = '1',  has_been_checked = '1',  instance_id = 1,  is_flapping = '0',  is_reachable = '1',  last_check = FROM_UNIXTIME(1403899271),  last_hard_state_change = FROM_UNIXTIME(1403898763),  last_state_change = FROM_UNIXTIME(1403898763),  last_time_critical = FROM_UNIXTIME(1403898703),  last_time_ok = FROM_UNIXTIME(1403899271),  last_time_warning = FROM_UNIXTIME(1403898733),  latency = '0',  long_output = '',  max_check_attempts = '3',  modified_service_attributes = '0',  next_check = FROM_UNIXTIME(1403899299),  normal_check_interval = '1',  notifications_enabled = '1',  output = 'PING OK - Packet loss = 0%, RTA = 28.91 ms',  passive_checks_enabled = '1',  percent_state_change = '11',  perfdata = 'pl=0%;10;20;0 rta=0.028908000999999999s;0.2;0.4;0',  problem_has_been_acknowledged = '0',  process_performance_data = '1',  retry_check_interval = '0.5',  scheduled_downtime_depth = '0',  service_object_id = 62,  should_be_scheduled = '1',  state_type = '1',  status_update_time = FROM_UNIXTIME(1403899271) WHERE service_object_id = 62

The script return diferent value (in miliseconds)

    PING OK - Packet loss = 0%, RTA = 28.91 ms|rta=28.912221ms;200.000000;400.000000;0.000000 pl=0%;10;20;0
